src/main/webapp/app/entities/participation/participation.route.ts

```diff
@@ -25,6 +27,15 @@ export const participationRoute: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
+    {
+        path: 'participation-submission/:participationId',
```

Comment:
Also here @maxr96: I think we would rather want `participation/:participationId/submissions`.